MATLAB/stalinsort.m

```diff
@@ -0,0 +1,7 @@
+function [orderedList] = stalinsort(list)
+
+orderedList = [list(1)];current=list(1);
+for(i=2:length(list))if(list(i)>current)
```

Comment:
what happens to the input `1 3 2 4 4 6`?

Reply:
It works with that input [1 3 2 4 4 6].
Or any of this examples
a) [1, 2, 3, 4, 5, 6, 7, 8, 9]
b) [9, 8, 7, 6, 5, 4, 3, 2, 1]
c) [1, 3, 2, 5, 4, 7, 6, 9, 8]
d) [1, 5, 2, 4]

with or without the commas but the square brackets are necessary

Reply:
but what is the output with repeated numbers?

Reply:
Repeated numbers are ignored (duplicates), just saving one of them.
![imagen](https://user-images.githubusercontent.com/42388031/67055183-0ad74c80-f115-11e9-89b1-b0ffaccf19b8.png)

Reply:
Ok! We need to keep the repeated numbers

Reply:
@sergiolibe can you fix this issue so we can merge the PR?

Reply:
Here's my fixed implementation. Now works on empty arrays too. Tested on MATLAB R2022a.

```matlab
function [orderedList] = stalinsort(list)
if isempty(list)
    orderedList = [];
else
    orderedList = [list(1)];
    current = list(1);
    for i=2:length(list)
        if(list(i) >= current)
            current = list(i);
            orderedList = [orderedList,current];
        end
    end
end
```